mautic/mautic
# 4.1 -> 4.x

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| Deprecations?                          | No
| BC breaks? (use the c.x branch)        | None

#### Description:

This PR merges the changes from 4.1.2 into the 4.x branch.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check that all is working as expected.


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10789"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
